### DISCUSSION TITLE
After deintsallation it is still showing the rainbows on variables

### DISCUSSION THREAD

**Discussion by dominik-gashi** [izhangzhihao/intellij-rainbow-brackets#2606] [Q&A] [answered]
I install it to try it out on WebStorm and then i disabled it and also deinstalled the plug and also restared WebStorm. After it is still showing the rainbow color in webstorm. What can I do?

**Comment by izhangzhihao** [comment, marked as answer]
Hi, you can disable it here: 
<img width="997" alt="image" src="https://user-images.githubusercontent.com/12044174/210359902-d08bc959-205b-40d3-be67-0c9cb907c823.png">

**Reply by dominik-gashi** [reply]
Thank you so much it worked.

**Comment by izhangzhihao** [comment]
Let's keep this thread open, in case someone else needs it.
